service/src/main/kotlin/no/nav/su/se/bakover/service/søknadsbehandling/SøknadsbehandlingServiceImpl.kt

```diff
@@ -348,9 +351,12 @@ internal class SøknadsbehandlingServiceImpl(
         ).map { iverksattBehandling ->
             when (iverksattBehandling) {
                 is Søknadsbehandling.Iverksatt.Innvilget -> {
+
                     søknadsbehandlingRepo.lagre(iverksattBehandling)
                     val vedtak = Vedtak.fromSøknadsbehandling(iverksattBehandling, utbetaling!!.id, clock)
                     vedtakRepo.lagre(vedtak)
+                    val kontrollsamtale = Kontrollsamtale.fraVedtak(vedtak)
```

Comment:
Hadde kanskje forventet at alle kontrollsamtalene ble planlagt i samme slengen? Hvordan er det tenkt at neste kontrollsamtale skal opprettes? En jobb som kaller `kallInn()` sakene i systemet med jevne mellomrom?

Reply:
Vi diskuterte litt frem og tilbake, men mener at vi havnet på følgende:

Problemet med å opprette alle kontrollsamtaler i samme slengen, er ved revurderinger etc som forskyver en kontrollsamtale. Vi tenker at det da blir mer komplisert å måtte legge inn logikk som sørger for at 'de andre kontrollsamtalene' også blir forskyvet. Vi har derfor tenkt at når schedule-jobben for den første kontrollsamtalen kjøres, så vil neste kontrollsamtale bli opprettet med eksisterende logikk (domain/src/main/kotlin/no/nav/su/se/bakover/domain/kontrollsamtale/Kontrollsamtale.kt), slik at vi unngår en haug med corner-cases

Reply:
Hvorfor blir kontrollsamtalene forskjøvet av en revurdering? Eller, i hvilke tilfeller?

Reply:
De blir vel ikke forskjøvet via revurderinger (opphør kan føre til at en planlagt kontrollsamtale ikke skal gjennomføres, men det er en annen sak), men fag har bestilt en mulighet for å *utsette* kontrollsamtaler, f.eks. ifm. at stønadsmottaker er i utlandet. Vi var innom tanken på å planlegge to samtaler ved stønadsperiode-innvilgelse, men har i utgangspunktet tenkt det er en fordel å slippe så mye som mulig bokholderi som vi da må holde i synk. Men det er noen fordeler med å opprette begge samtalene ved start også, så vi kommer nok til å tenke litt mer på det.

Reply:
Jeg renger med at den eneste påvirkningen en revurdering eventuelt kunne hatt er at den kan føre til at en planlagt kontrollsamtale skal destrueres som følge av at vi f.eks har opphørt ytelsen? Dette var et typisk use-case jeg hadde i i hodet mitt når jeg foreslo at det var `Sak` som burde stå for "orkestreringen" av kontrollsamtalene. Mye av de samme tankene tror jeg holder vann for det å skulle kunne utsette 1 eller flere fremtidige kontrollsamtaler.